charts/tfy-llm-gateway-infra/values.yaml

```diff
@@ -222,7 +221,5 @@ clickhouse:
     storageClassName: ''
     size: 10Gi
   nats:
-    url: >-
```

Comment:
wouldn't it be better to keep this configurable?

Reply:
hmm yeah can do